src/transformers/models/tapas/configuration_tapas.py

```diff
@@ -87,41 +87,49 @@ class TapasConfig(PretrainedConfig):
             aggregation (WTQ, WikiSQL). If the total mass of the aggregation probabilities (excluding the "NONE"
             operator) is higher than this hyperparameter, then aggregation is predicted for an example.
-        answer_loss_cutoff: (:obj:`float`, `optional`):
+        answer_loss_cutoff (:obj:`float`, `optional`):
             Ignore examples with answer loss larger than cutoff.
-        max_num_rows: (:obj:`int`, `optional`, defaults to 64):
+        max_num_rows (:obj:`int`, `optional`, defaults to 64):
             Maximum number of rows.
-        max_num_columns: (:obj:`int`, `optional`, defaults to 32):
+        max_num_columns (:obj:`int`, `optional`, defaults to 32):
             Maximum number of columns.
-        average_logits_per_cell: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+        average_logits_per_cell (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to average logits per cell.
-        select_one_column: (:obj:`bool`, `optional`, defaults to :obj:`True`):
+        select_one_column (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether to constrain the model to only select cells from a single column.
-        allow_empty_column_selection: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+        allow_empty_column_selection (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to allow not to select any column.
-        init_cell_selection_weights_to_zero: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+        init_cell_selection_weights_to_zero (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to initialize cell selection weights to 0 so that the initial probabilities are 50%.
-        reset_position_index_per_cell: (:obj:`bool`, `optional`, defaults to :obj:`True`):
+        reset_position_index_per_cell (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether to restart position indexes at every cell (i.e. use relative position embeddings).
-        disable_per_token_loss: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+        disable_per_token_loss (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to disable any (strong or weak) supervision on cells.
+        aggregation_labels (:obj:`Dict[int, label]`, `optional`):
+            The aggregation labels used to aggregate the results. For example, the WTQ models have the following
+            aggregation labels: :obj:`{0: "NONE", 1: "SUM", 2: "AVERAGE", 3: "COUNT"}`
+        no_aggregation_label_index (:obj:`int`, `optional`):
+            If the aggregation labels are defined and one of these labels represents "No aggregation", this should be
+            set to its index. For example, the WTQ models have the "NONE" aggregation label at index 0, so that value
+            should be set to 0 for these models.
+
```

Comment:
If these changes are accepted, these two attributes should be added to the configuration like it was done on this [tiny model](https://huggingface.co/lysandre/tiny-tapas-random-wtq/blob/main/config.json). This allows the pipeline to generate the correct aggregator and to ignore it when it shouldn't be accounted for.

Reply:
good for me!

Reply:
If this is added, does that mean the `config.json` of all models on the hub need to be updated?

Reply:
Yes, I think Patrick has a bash script made especially for this!